scripts/build/Platform/Linux/deploy_cdk_applications.sh

```diff
@@ -9,7 +9,7 @@
 # Deploy the CDK applications for AWS gems (Linux only)
 
 SOURCE_DIRECTORY=$PWD
-PATH=$SOURCE_DIRECTORY/python:$PATH
+PATH=$SOURCE_DIRECTORY/python/runtime/$PYTHON_RUNTIME/python/bin:$PATH
```

Comment:
It seems strange to call out a specific python binary here when we do not control its install. Is there a reason we can't use python_linux.sh ?

Reply:
We can use python.sh explicitly for CLI commands like "python.sh -m pip install -r requirements.txt". However, CDK calls "python" directly when it deploy resources which causes it to use system default Python instead of O3DE Python.